src/main/thrift/parquet.thrift

```diff
@@ -373,6 +459,56 @@ struct JsonType {
 struct BsonType {
 }
 
+/**
+ * Physical type and encoding for the geometry type.
+ */
+enum GeometryEncoding {
+  /**
+   * Allowed for physical type: BYTE_ARRAY.
+   *
+   * Well-known binary (WKB) representations of geometries. It supports 2D or
```

Comment:
Shall we specify here that this should be ISO WKB (and not "extended WKB", using the terminology of https://libgeos.org/specifications/wkb/). 

The text in the geoparquet spec about this: https://github.com/opengeospatial/geoparquet/blob/main/format-specs/geoparquet.md#wkb

Reply:
This was mentioned in this spec:

```
This is borrowed from `geometry_types` column metadata of GeoParquet [1]
   * except that values in the list are WKB (ISO variant) integer codes [2]. Table
   * below shows the most common geometry types and their codes:
```